modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/java/com/liferay/portal/search/tuning/rankings/web/internal/upgrade/v1_0_0/RankingUpgradeProcess.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.search.tuning.rankings.web.internal.upgrade.v1_0_0;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.search.tuning.rankings.web.internal.constants.ResultRankingsConstants;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Almir Ferreira
+ */
+public class RankingUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		try (PreparedStatement preparedStatement1 = connection.prepareStatement(
+				"select jsonStorageEntryId, valueString from " +
+					"jsonStorageEntry where key_ = ?");
```

Comment:
i think we should add a clause here in the select to only target rows where classNameId is for com.liferay.portal.search.tuning.rankings.web.internal.index.Ranking

the json storage table can store data for anything, so its not enough to just target the key_, otherwise we could potentially upgrade some unrelated data

Reply:
Hey @BryanEngler, I have already sent a commit with your suggestion.

Reply:
nice, looks good, thanks